dojo/jira_link/helper.py

```diff
@@ -874,6 +889,16 @@ def jira_get_issue(jira_project, issue_key):
         return None
 
 
+# Return a list of jira issue in json format.
+def get_epic_issues(engagement):
+    instance = get_jira_instance(engagement)
+    jira = get_jira_connection(instance)
+    epic_id = get_jira_issue_key(engagement)
+    url = instance.url.strip('/') + '/rest/agile/1.0/epic/' + epic_id + '/issue'
+    response = jira._session.get(url).json()
+    return response.get('issues', [])
+
+
```

Comment:
is this only used in the tests? because in that case it would be better placed in dojo_test_case.

Reply:
Sure can do.